plugins/modules/network_settings_intent.py

```diff
@@ -105,23 +121,23 @@
           ipv4_subnet:
             description: IPv4 Subnet address, example 175.175.0.0.
             type: str
-          ipv4TotalHost:
+          ipv4_total_host:
             description: IPv4 total host is required when ipv4_prefix value is false.
             type: int
           ipv6_address_space:
             description: >
               If the value is false only ipv4 input are required, otherwise both
               ipv6 and ipv4 are required.
             type: bool
-          ipv6DhcpServers:
+          ipv6_dhcp_servers:
             description: IPv6 format dhcp server as input example 2001 db8 1234.
             elements: str
             type: list
-          ipv6DnsServers:
+          ipv6_dns_servers:
             description: IPv6 format dns server input example 2001 db8 1234.
             elements: str
             type: list
-          ipv6GateWay:
+          ipv6_gateway:
             description: Gateway ip address details, example 2001 db8 85a3 0 100 1.
```

Comment:
Provides the gateway's IPv6 address; for example, an address similar to "2001:0db8:0123:4567:89ab:cdef:0003:0003"